nncf/quantization/algorithms/weight_compression/algorithm.py

```diff
@@ -369,7 +369,7 @@ def apply(
         self._set_weight_compression_config(ratio_defining_params, model, graph, activations)
         nncf_logger.info(self._get_bitwidth_distribution_str(all_weight_params, ratio_defining_params))
 
-        if self._awq and activations is not None and self._mode != CompressWeightsMode.E2M1:
```

Comment:
Because it's already checked here: https://github.com/openvinotoolkit/nncf/blob/develop/nncf/quantization/quantize_model.py#L518-L524
Unit tests check that:
https://github.com/openvinotoolkit/nncf/pull/3002/files#diff-223ea638f7751f7c0c3e8f867ec9c8c132a3ccd62a9dcea2a5d158836c71c222R730-R741